docs/flashbots-mev-share/searchers/getting-started.mdx

```diff
@@ -9,20 +9,20 @@ To start searching on MEV-Share, you will first need to connect to a MEV-Share N
 
 ## Connect to MEV-Share Node
 
-Flashbots runs a MEV-Share Node on Ethereum mainnet. The MEV-Share Node has endpoints to receive transactions and bundles, as well as an SSE event stream endpoint which dispatches pending events/transactions to searchers.
+Flashbots operates an MEV-Share Node on the Ethereum mainnet. This node provides endpoints for user transactions and searcher bundles. Additionally, it offers an SSE event stream endpoint, broadcasting pending events/transactions to searchers.
```

Comment:
Nit: For consistency, I usually capitalize "Node" even when not prefixed by "MEV-Share".

Reply:
Can take or leave this suggestion but thought last sentence could be clearer: "It also offers an SSE event stream endpoint which shares information about pending transactions/bundles with searchers."